client/src/containers/Workspace/Workspace.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Container, InputField, SidebarList } from '../../components';
+
+const Workspace = ({userName, ...props}) => {
```

Comment:
Consider changing the name `Workspace` to `Workspaces`, from the folder level. This would help keep the semantics consistent with what the component is doing.